deploy/namespace.yaml

```diff
@@ -4,3 +4,6 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/enforce: privileged
```

Comment:
Doesn't seem to be working for me in 4.12. I tried yours and this:
diff --git a/deploy/namespace.yaml b/deploy/namespace.yaml
index 37284785..dd8a4ed7 100644
--- a/deploy/namespace.yaml
+++ b/deploy/namespace.yaml
@@ -4,3 +4,8 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.24
+    security.openshift.io/scc.podSecurityLabelSync: "false"
However the config daemon is not created.
Only when I do:
oc label ns openshift-sriov-network-operator pod-security.kubernetes.io/audit=privileged pod-security.kubernetes.io/enforce=privileged pod-security.kubernetes.io/warn=privileged security.openshift.io/scc.podSecurityLabelSync=false
Then I see the sriov-network-config-daemon being created.

Reply:
Sorry I forgot to delete the namespace with:
```
oc delete namespace openshift-sriov-network-operator
```
However only this works for me. Your changes don't seem to work. Most notably the podSecurirtyLabelSync has to be false for me:
```
diff --git a/deploy/namespace.yaml b/deploy/namespace.yaml
index 37284785..dd8a4ed7 100644
--- a/deploy/namespace.yaml
+++ b/deploy/namespace.yaml
@@ -4,3 +4,8 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/audit: privileged
+    pod-security.kubernetes.io/warn: privileged
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.24
+    security.openshift.io/scc.podSecurityLabelSync: "false"
```

Reply:
please take a look I change the labelSync to false.

the namespace you use start with `openshift-` :)

Reply:
Seems to work properly now! Thanks Seb:
```
[root@wsfd-advnetlab28 sriov-network-operator]# git diff
diff --git a/deploy/namespace.yaml b/deploy/namespace.yaml
index 37284785..81303160 100644
--- a/deploy/namespace.yaml
+++ b/deploy/namespace.yaml
@@ -4,3 +4,6 @@ metadata:
   name: $NAMESPACE
   labels:
     name: $NAMESPACE
+    pod-security.kubernetes.io/enforce: privileged
+    pod-security.kubernetes.io/enforce-version: v1.24
+    security.openshift.io/scc.podSecurityLabelSync: "false"
[root@wsfd-advnetlab28 sriov-network-operator]# oc get pods -n openshift-sriov-network-operator
NAME                                      READY   STATUS    RESTARTS   AGE
network-resources-injector-4qxdb          1/1     Running   0          27s
network-resources-injector-jfjs4          1/1     Running   0          27s
network-resources-injector-k4jj8          1/1     Running   0          27s
operator-webhook-nzw8v                    1/1     Running   0          27s
operator-webhook-sm6zd                    1/1     Running   0          27s
operator-webhook-t69w5                    1/1     Running   0          27s
sriov-network-config-daemon-7htv9         3/3     Running   0          27s
sriov-network-config-daemon-8mk6r         3/3     Running   0          27s
sriov-network-config-daemon-ffjgg         3/3     Running   0          27s
sriov-network-config-daemon-gdmrc         3/3     Running   0          27s
sriov-network-config-daemon-lwqqd         3/3     Running   0          27s
sriov-network-operator-7cdf755677-kfscn   1/1     Running   0          36s
[root@wsfd-advnetlab28 sriov-network-operator]# oc describe project openshift-sriov-network-operator
Name:			openshift-sriov-network-operator
Created:		49 seconds ago
Labels:			kubernetes.io/metadata.name=openshift-sriov-network-operator
			name=openshift-sriov-network-operator
			pod-security.kubernetes.io/enforce=privileged
			pod-security.kubernetes.io/enforce-version=v1.24
			security.openshift.io/scc.podSecurityLabelSync=false
Annotations:		kubectl.kubernetes.io/last-applied-configuration={"apiVersion":"v1","kind":"Namespace","metadata":{"annotations":{},"labels":{"name":"openshift-sriov-network-operator","pod-security.kubernetes.io/enforce":"privileged","pod-security.kubernetes.io/enforce-version":"v1.24","security.openshift.io/scc.podSecurityLabelSync":"false"},"name":"openshift-sriov-network-operator"}}
			
			openshift.io/sa.scc.mcs=s0:c26,c10
			openshift.io/sa.scc.supplemental-groups=1000670000/10000
			openshift.io/sa.scc.uid-range=1000670000/10000
Display Name:		<none>
Description:		<none>
Status:			Active
Node Selector:		<none>
Quota:			<none>
Resource limits:	<none>
```